modules/javafx.controls/src/main/java/javafx/scene/control/TableView.java

```diff
@@ -395,6 +395,92 @@
         }
     };
 
+    /**
+     * A policy that tries to adjust other columns in order to fit the table width.
+     * <p>
```

Comment:
For all of the constrained policies, I would say `A resize policy that adjusts...`. I think the word "resize" is important in the first sentence, and I don't think you need to say "tries", since it will adjust it, subject to the minimum size of the columns being adjusted (and the details of what happens when it hits the limit are explained later). Also, I don't think the `<p>` is really needed after the first sentence.